btrfs.go

```diff
@@ -383,29 +383,29 @@ func SubvolDelete(path string) error {
 func openSubvolDir(path string) (*os.File, error) {
 	fp, err := os.Open(path)
 	if err != nil {
-		return nil, errors.Wrapf(err, "opening %v as subvolume failed", path)
+		return nil, fmt.Errorf("opening %v as subvolume failed: %w", path, err)
 	}
 
 	return fp, nil
 }
 
 func isStatfsSubvol(statfs *syscall.Statfs_t) error {
 	if int64(statfs.Type) != int64(C.BTRFS_SUPER_MAGIC) {
-		return errors.Errorf("not a btrfs filesystem")
+		return fmt.Errorf("not a btrfs filesystem")
 	}
 
 	return nil
 }
 
 func isFileInfoSubvol(fi os.FileInfo) error {
 	if !fi.IsDir() {
-		errors.Errorf("must be a directory")
+		errors.New("must be a directory")
```

Comment:
looks like there was a mistake here with no `return`; CI now caught it:
```
Error: ./btrfs.go:402:13: result of errors.New call not used
```

Reply:
Done